parsing/parser.mly

```diff
@@ -2391,6 +2395,11 @@ label_let_pattern:
       { let lab, pat = x in
         lab,
         mkpat ~loc:$sloc (Ppat_constraint (pat, cty)) }
+  | x = label_var COLON
+          cty = mktyp (vars = typevar_list DOT ty = core_type { Ptyp_poly(vars, ty) })
+      { let lab, pat = x in
+        lab,
+        mkpat ~loc:$sloc (Ppat_constraint (pat, cty)) }
```

Comment:
Wouldn't it be nicer to introduce a new parsing rule here ?
```ocaml
maybe_core_type:
  | cty = core_type { cty }
  | poly_case
```

It could be used here and in the `let_pattern` case bellow

Reply:
This sounds like  'poly_type' which is already defined.